solutions/standard/variables.tf

```diff
@@ -53,19 +53,19 @@ variable "key_protect_allowed_network" {
 
 variable "resource_tags" {
   type        = list(string)
-  description = "Optional list of tags to be added to the Key Protect instance. Only used if not supplying an existing KMS instance."
+  description = "Optional list of tags to be added to the Key Protect instance. Only used if not supplying an existing Key Protect instance."
```

Comment:
`Key Protect or Hyper Protect Crypto Services instance.`

Reply:
The variable is literally `key_protect_allowed_network` -- how can HPCS be relevant here?